srcpkgs/volctl/template

```diff
@@ -1,13 +1,22 @@
 # Template file for 'volctl'
 pkgname=volctl
-version=0.8.0
-revision=2
+version=0.8.2
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-gobject desktop-file-utils"
-short_desc="Lightweight volume control for pulseaudio that sits in your systray"
+depends="python3-gobject desktop-file-utils python3-cairo python3-yaml python3-click"
+checkdepends="${depends} dbus pulseaudio gtk+3 xvfb-run"
```

Comment:
This makes me think that maybe it should depend on `gtk+3` itself?

Reply:
Moved.